cyral/model_integration_logging.go

```diff
@@ -0,0 +1,258 @@
+			Description:   "Represents the configuration data required for the ELK stack log management system.",
+			Optional:      true,
+			Type:          schema.TypeSet,
+			ConflictsWith: []string{CloudWatchKey, DatadogKey, SplunkKey, SumoLogicKey, FluentbitKey},
+			MaxItems:      1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"es_url": {
+						Description: "Elasticsearch URL.",
+						Required:    true,
+						Type:        schema.TypeString,
+					},
+					"kibana_url": {
+						Description: "Kibana URL.",
+						Optional:    true,
+						Type:        schema.TypeString,
+					},
+					"es_credentials": {
+						Description: "Credentials used to authenticate with Elastic Search",
+						Optional:    true,
+						Type:        schema.TypeSet,
+						MaxItems:    1,
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"username": {
+									Description: "Elasticsearch username.",
+									Required:    true,
+									Type:        schema.TypeString,
+								},
+								"password": {
+									Description: "Elasticsearch password.",
+									Required:    true,
+									Sensitive:   true,
+									Type:        schema.TypeString,
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+
+		"splunk": {
+			Description:   "Represents the configuration data required for the Splunk log management system.",
+			Optional:      true,
+			Type:          schema.TypeSet,
+			ConflictsWith: []string{CloudWatchKey, DatadogKey, ElkKey, SumoLogicKey, FluentbitKey},
+			MaxItems:      1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"hostname": {
+						Description: "Splunk hostname.",
+						Required:    true,
+						Type:        schema.TypeString,
+					},
+					"hec_port": {
+						Description: "Splunk HTTP Event Collector (HEC) port.",
+						Required:    true,
+						Type:        schema.TypeString,
+					},
+					"access_token": {
+						Description: "Splunk access token.",
+						Required:    true,
+						Sensitive:   true,
+						Type:        schema.TypeString,
+					},
+					"index": {
+						Description: "Splunk index which logs should be indexed to.",
+						Optional:    true,
+						Type:        schema.TypeString,
+					},
+					"use_tls": {
+						Description: "Whether or not to use TLS.",
+						Optional:    true,
+						Type:        schema.TypeBool,
+					},
+				},
+			},
+		},
+
+		"sumo_logic": {
+			Description:   "Represents the configuration data required for the Sumo Logic log management system.",
+			Optional:      true,
+			Type:          schema.TypeSet,
+			ConflictsWith: []string{CloudWatchKey, DatadogKey, ElkKey, SplunkKey, FluentbitKey},
+			MaxItems:      1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"address": {
+						Description: "Sumo Logic HTTP collector address. A full URL is expected",
+						Required:    true,
+						Type:        schema.TypeString,
+					},
+				},
+			},
+		},
+
+		"fluentbit": {
```

Comment:
I would change the name to `fluent_bit` here (and everywhere else it is defined as `fluentbit`).